terraform/kevsync_failure_sns_topic.tf

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------------------
+# Create the SNS topic that allows email to be sent for CloudWatch
+# alarms related to kevsync failures on the database instances.
+# ------------------------------------------------------------------------------
+
+resource "aws_sns_topic" "kevsync_failure_alarm" {
+  name         = "kevsync-failure-alarms"
+  display_name = "kevsync_failure_alarms"
+}
+
+resource "aws_sns_topic_subscription" "kevsync_failure_email" {
+  for_each = toset([
+    "cyberdirectives@cisa.dhs.gov",
+    "vulnerability@cisa.dhs.gov",
```

Comment:
This will send failure emails to those two addresses in addition to `cisa-cool-group+cyhy@trio.dhs.gov`, correct?

Reply:
Note that we list `aws_sns_topic.cloudwatch_alarm.arn` in the `alarm_actions`, `insufficient_data_actions`, and `ok_actions` attributes of `aws_cloudwatch_metric_alarm.kevsync_failure`.  Thus `cisa-cool-group+cyhy@trio.dhs.gov` will get emails for all state transitions.